pkg/quota/evaluator/core/pods_test.go

```diff
@@ -266,6 +282,22 @@ func TestPodEvaluatorUsage(t *testing.T) {
 				generic.ObjectCountQuotaResourceNameFor(schema.GroupResource{Resource: "pods"}): resource.MustParse("1"),
 			},
 		},
+		"container extended resources": {
+			pod: &api.Pod{
+				Spec: api.PodSpec{
+					Containers: []api.Container{{
+						Resources: api.ResourceRequirements{
+							Requests: api.ResourceList{api.ResourceName("example.com/dongle"): resource.MustParse("3")},
+						},
+					}},
+				},
+			},
+			usage: api.ResourceList{
+				api.ResourceName("requests.example.com/dongle"): resource.MustParse("3"),
+				api.ResourcePods:                                resource.MustParse("1"),
+				generic.ObjectCountQuotaResourceNameFor(schema.GroupResource{Resource: "pods"}): resource.MustParse("1"),
+			},
+		},
 		"init container maximums override sum of containers": {
 			pod: &api.Pod{
```

Comment:
Maybe extend this case a bit to also include extended resource requests to make sure we calculate the max resource requests for ER correctly?